crates/ide/src/highlight_related.rs

```diff
@@ -281,99 +281,95 @@ fn highlight_references(
     }
 }
 
-// If `file_id` is None,
-pub(crate) fn highlight_exit_points(
+fn hl_exit_points(
```

Comment:
It looks like a ton of stuff happened here, but nothing is actually changed and the function just moved out, so it can be called from `highlight_yield_points`